app/config/config.yml

```diff
@@ -171,3 +171,15 @@ fos_oauth_server:
     auth_code_class:     Wallabag\ApiBundle\Entity\AuthCode
     service:
         user_provider: fos_user.user_manager
+
+scheb_two_factor:
+    trusted_computer:
+        enabled: true
```

Comment:
I was thinking about these config to enable two factor.
I think they should be in `parameters.yml` so people won't have to update `config.yml` and keep a *modified file* which can be problematic when updating wallabag.

Also, the sender_email should be in parameters.yml